editor.go

```diff
@@ -286,6 +287,14 @@ func (e *Editor) configure() {
 		e.drawStatusline = true
 	}
 
+	var drawTabline interface{}
+	e.nvim.Var("gonvim_draw_tabline", &drawTabline)
+	if isZero(drawTabline) {
+		e.drawTabline = false
+	} else {
+		e.drawTabline = true
+	}
+
```

Comment:
o["ext_tabline"] = true can't be added if draTabline is false, so you need to move editor.configure() above attachUI

Reply:
I fixed it so o["ext_tabline"] use the drawTabline flag.